### DISCUSSION TITLE
Reporting metrics for each vU

### DISCUSSION THREAD

**Discussion by vikramjb** [artilleryio/artillery#2523] [Artillery]
I have created a script for testing a particular process flow that test for 10 users and gives me metrics about the TTFB an others and how long each "step" took. What I am missing or not sure where to find is metrics for how long each "step" too for each virtual user. Is that behind a flag or is not available or is that something that can be derived?

**Comment by hassy** [comment]
hi @vikramjb 👋 Artillery does not provide per-VU metrics in its default text report (the amount of data would quickly become impossible to read for a load test). You can get that information if you set up tracing with one of the supported observability platforms via [`publish-metrics`](https://www.artillery.io/docs/reference/extensions/publish-metrics). There's a short demo here: https://www.artillery.io/blog/end-to-end-tracing-for-playwright-tests

**Reply by vikramjb** [reply]
Hi @hassy 👋 

Thank you for the quick response. I will try out honeycomb with playwright and get back if there are any issues.
